hass-addon-sunsynk-multi/config.yaml

```
@@ -44,11 +45,12 @@ options:
   DEBUG: 0
   DEBUG_DEVICE: "/dev/ttyAMA0"
 schema:
-  DRIVER: list(pymodbus|umodbus)
+  DRIVER: list(pymodbus|umodbus|solarman)
   INVERTERS:
     - SERIAL_NR: str
       HA_PREFIX: str
       MODBUS_ID: int(1,16)
+      DONGLE_SERIAL_NUMBER: str
```

This is [optional](https://developers.home-assistant.io/docs/add-ons/configuration/#options--schema)

```suggestion
      DONGLE_SERIAL_NUMBER: str?
```

---

I've made this change